src/app/modules/account/pages/details/components/contact-information/contact-information.component.html

```diff
@@ -0,0 +1,31 @@
+<ion-card>
+  <ion-card-header>
+    <ion-card-title>Contact information</ion-card-title>
+  </ion-card-header>
+  <ion-card-content>
+    <ion-grid>
+      <ion-row>
+        <ion-col size="6">
+          <div>
+            <h2>Phone</h2>
+            <p>{{ getPhoneNumber() }}</p>
+          </div>
+          <div>
+            <h2>Email address</h2>
+            <p>{{ account?.email }}</p>
+          </div>
+          <div>
+            <h2>Mailing address</h2>
+            <p>{{ account?.contactInformation?.mailingAddress ?? '-' }}</p>
```

Comment:
I think for this we want to use the `account?.contactInformation?.addresses` array.  you can probably just display the first address in the array or iterate over them.